executors/rust/1.3/Cargo.toml

```diff
@@ -13,11 +13,16 @@ serde = "1.0.171"
 serde_json = "1.0.100"
 rustc_version_runtime = "0.1.*"
     
-icu = { version = "~1.3", features = ["serde", "icu_compactdecimal", "icu_displaynames", "compiled_data"] }
+icu = { version = "1.3.2", features = ["serde", "icu_compactdecimal",
```

Comment:
Issue: You need `~` dependencies because otherwise cargo will install the latest version, not 1.3